README.md

```diff
@@ -26,21 +26,21 @@ steampipe query
 Run a query:
 ```sql
 select
```

Comment:
I'm sure this example works, but is there something potentially more interesting we can use as an example here and in the README? For instance, something around showing builds, or showing interesting team or user information?